### PR TITLE
Fix a typo in CHANGELOG.md

### PR DESCRIPTION
**Pull request by hs0210**
Fix a typo "the the" -> "the"

Signed-off-by: Hu Shuai <hus.fnst@cn.fujitsu.com>

## Which problem is this PR solving?
- 

## Short description of the changes
-

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ Changes by Version
         If you don't run the migration script, nothing will break, the system will used the old table 
         `operation_names` and set empty `spanKind` in the response.  
         Steps to get the updated functionality:
-        1.  You will need to run below command on the host you can use `cqlsh` to connect the the cassandra contact
+        1.  You will need to run below command on the host you can use `cqlsh` to connect the cassandra contact
```
Comment (yurishkuro) [inline CHANGELOG.md]:
```suggestion
        1.  You will need to run the command below on the host where you can use `cqlsh` to connect to Cassandra:
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -58,7 +58,7 @@ Changes by Version
...
          point
```
Comment (yurishkuro) [inline CHANGELOG.md]: remove this line

Comment (hs0210) [reply CHANGELOG.md]: So should I remove this line or change it? I don't think removing it is a right approach, it maybe cause some confusion.

Comment (yurishkuro) [reply CHANGELOG.md]: if you apply my previous suggestion, this standalone word "point" should be removed

**Pull request merged**